great_expectations/data_context/store/ge_cloud_store_backend.py

```diff
@@ -34,7 +34,8 @@ class GeCloudRESTResource(str, Enum):
 
 class GeCloudStoreBackend(StoreBackend, metaclass=ABCMeta):
     PAYLOAD_ATTRIBUTES_KEYS: Dict[GeCloudRESTResource, str] = {
-        GeCloudRESTResource.CONTRACT: "checkpoint_config",
+        GeCloudRESTResource.CHECKPOINT: "checkpoint_config",
+        GeCloudRESTResource.CONTRACT: "checkpoint_config",  # Deprecated
```

Comment:
We need to keep this around as our E2E tests rely on old behavior - leaving a generic comment here but just wanted to flag for discussion before we move forward.